src/main/resources/application.yml

```diff
@@ -130,6 +130,10 @@ folio:
         concurrency: ${KAFKA_CLASSIFICATION_TYPE_CONCURRENCY:1}
         topic-pattern: (${folio.environment}\.)(.*\.)inventory\.classification-type
         group-id: ${folio.environment}-mod-search-classification-type-group
+      call-number-type:
+        concurrency: ${KAFKA_CALL_NUMBER_TYPE_CONCURRENCY:1}
+        topic-pattern: (${folio.environment}\.)(.*\.)inventory\.call-number-type
+        group-id: ${folio.environment}-mod-search-call-number-type-group
```

Comment:
I propose to use same listener as for classification types because logic is same. Just update pattern
`topic-pattern: (${folio.environment}\.)(.*\.)inventory\.(classification-type|call-number-type)`
And also update name of the listener.